pyramid_assemble.py

```diff
@@ -236,8 +239,17 @@ def main():
 
         print()
 
+    if dtype == np.uint32 and is_mask:
```

Comment:
Can you make this a nested if for the `is_mask` check so it can print a more specific error message? How about "uint32 images are only supported in --mask mode. Please contact the authors if you need support for intensity-based uint32 images.". print then sys.exit(1).

Reply:
I've added the nested if and used your suggested error message.